client/.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
+	"printWidth": 100,
+	"tabWidth": 2,
```

Comment:
Maybe it is better to make a smaller tabs?

Reply:
2 spaces for tabs is a standard, we can't do less, we just get a simple space